Interoperability/python/qsharp/__main__.py

```diff
@@ -12,8 +12,9 @@
 from qsharp.tomography import single_qubit_process_tomography
 from Microsoft.Quantum.Primitive import I
 
-qsim = qsharp.QuantumSimulator()
-noise_channel = qsim.get(I)
-estimation_results = single_qubit_process_tomography(qsim, noise_channel, n_measurements=2000)
+QSIM = qsharp.QuantumSimulator()
+NOISE_CHANNEL = QSIM.get(I)
+ESTIMATION_RESULTS = single_qubit_process_tomography(
```

Comment:
The estimation results are definitely not a constant, and should not be treated as such.